docs/sources/reference/components/loki/loki.source.file.md

```diff
@@ -51,12 +51,8 @@ If not set, it defaults to UTF-8.
 You can use the `tail_from_end` argument when you want to tail a large file without reading its entire content.
 When set to true, only new logs are read, ignoring the existing ones.
 
-{{< admonition type="note" >}}
 The `legacy_positions_file` argument is used when you are transitioning from legacy. The legacy positions file is converted to the new format.
 This operation only occurs if the new positions file doesn't exist and the `legacy_positions_file` is valid.
-If you add any labels before `loki.source.file`, then the positions file conversion won't work.
-The legacy positions file didn't have a concept of labels in the positions file, so the conversion assumes no labels.
-{{< /admonition >}}
```

Comment:
IIUC, ignoring the labels is assuming that a file is only present once in `targets`? And the reason for tracking labels in Alloy's loki.source.file is that a file may be present more than once, with different labels? If that's the case should we check if a file is present more than once and return a warning?

There must be a reason for tracking the labels, so I'm just wondering what could possibly break if we don't.

Reply:
Yeah I have not found the reason for also tacking labels. I can try to figure out why did was done. It is possible to tail the same file if we have several targets with different labels but the same __path__.